sdk/storage_datalake/README.md

```diff
@@ -8,5 +8,5 @@ To set this crate as a dependency, add this to your Cargo.toml
 
 ```toml
 [dependencies]
-azure_storage_datalake = { version = "0.1.0", git = "https://github.com/Azure/azure-sdk-for-rust" }
+azure_storage_datalake = { version = "0.1.1", git = "https://github.com/Azure/azure-sdk-for-rust" }
```

Comment:
Should be `azure_storage_datalake ="0.1"`.

Reply:
As discussed, removing Usage section altogether since users need to decide how to specify dependency versions.